cinema/models.py

```diff
@@ -0,0 +1,13 @@
+from django.db import models
+
+
+class Movie(models.Model):
+    title = models.CharField(max_length=255, blank=False, null=False)
+    description = models.TextField(blank=True)
+    duration = models.TimeField(blank=True, null=True)
+
+    class Meta:
+        verbose_name_plural = "movies"
```

Comment:
I'm not sure you really need it, because django creates plural right by itself, but okey